src/org/labkey/targetedms/view/figuresOfMerit.jsp

```diff
@@ -98,7 +98,8 @@
         <td>
             <% if (!bean.getMinimize()) { %>
                 <div class="export-icon" data-toggle="tooltip" title="Export to Excel">
-                    <a id="targetedms-fom-export" href="javascript:exportExcel()"><i class="fa fa-file-excel-o"></i> Export to Excel</a>
+                    <%= iconLink("fa fa-file-excel-o", "Export to Excel", null).href("javascript:exportExcel()")%>
+                    <%= link("Export to Excel").id("targetedms-fom-export").href("javascript:exportExcel()")%>
```

Comment:
Note to the code reviewer - link("Export to Excel").id("targetedms-fom-export").href("javascript:exportExcel()").iconCls("fa fa-file-excel-o") - removes the text - "Export to Excel".. so had to split into two elements.

Reply:
More standard would be `.onClick("exportExcel();")` instead of using href.